packages/presentation/lib/screen/movie_details/widgets/details_tab_bar.dart

```diff
@@ -30,22 +30,23 @@ class DetailsTabBar extends StatefulWidget {
 class _DetailsTabBarState extends State<DetailsTabBar>
     with SingleTickerProviderStateMixin {
   late TabController tabController;
-  int currentIndex = 0;
+  int _currentIndex = 0;
+  final int _tabControllerLength = 3;
```

Comment:
why _tabControllerLength is final, but  _currentIndex not?
and if these variables immutable they must be static const

Reply:
currentIndex changes, that's why it can't be final or static const